lib/android/build.gradle

```diff
@@ -38,7 +38,5 @@ dependencies {
   compile 'com.android.support:support-annotations:25.2.0'
   compile 'com.android.support:design:25.2.0'
   compile 'com.fasterxml.jackson.core:jackson-databind:2.8.3'
-  compile 'com.facebook.fresco:fresco:0.11.0'
```

Comment:
Done. can we remove fresco dependencies? because react native has been dependent on it: [build.gradle#L282](https://github.com/facebook/react-native/blob/master/ReactAndroid/build.gradle#L282), if it's good enough I would be happy to submit a PR.